app/controllers/idv/link_sent_controller.rb

```diff
@@ -0,0 +1,121 @@
+module Idv
+  class LinkSentController < ApplicationController
+    include IdvSession
+    include IdvStepConcern
+    include StepIndicatorConcern
+    include StepUtilitiesConcern
+
+    before_action :render_404_if_link_sent_controller_disabled
+    before_action :confirm_two_factor_authenticated
+    before_action :confirm_upload_step_complete
+    before_action :confirm_document_capture_needed
+
+    def show
+      analytics.idv_doc_auth_link_sent_visited(**analytics_arguments)
+
+      Funnel::DocAuth::RegisterStep.new(current_user.id, sp_session[:issuer]).
+        call('link_sent', :view, true)
+
+      render :show, locals: extra_view_variables
+    end
+
+    def update
```

Comment:
Is this action ever called? I'm testing locally and I see:

1. Enter phone number on desktop, get redirected to the "We sent a message to your phone" screen
2. Desktop starts polling `/verify/doc_auth/link_sent/poll`, receives `202 Accepted` which it [interprets](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/document-capture-polling/index.ts#L125-L127) as "keep polling"
3. Upload documents on phone
4. Desktop starts to receive `200 OK` on polling, which it interprets as "all done, submit the `<form>` on the page with the class `link-sent-continue-button-form`".
5. That form has `action="/verify/ssn"`, so the user goes the SSN page
6. The SSN page is looking for `flow_session['pii_from_doc']`, but not finding it, and redirecting the user back to the link sent page

I'm not seeing where we actually do an `update` in all this--we _might_ need to call `handle_document_verification_success` in the `show` action (or have the SSN screen look at the actual document capture session result rather than looking in `flow_session`)

Reply:
This is now fixed, with @jmhooper's help.